src/sdk/models/smil-model.js

```diff
@@ -0,0 +1,760 @@
+//  LauncherOSX
```

Comment:
[And it is there](https://github.com/dariocravero/readium-js/blob/master/src/sdk/models/smil/smil-node.js). The public interface is provided by `Smil` and not by the node, the node is required by more complex nodes such as [MediaNode](https://github.com/dariocravero/readium-js/blob/master/src/sdk/models/smil/media-node.js#L1) which in turn construct both [AudioNode](https://github.com/dariocravero/readium-js/blob/master/src/sdk/models/smil/audio-node.js#L1) and [TextNode](https://github.com/dariocravero/readium-js/blob/master/src/sdk/models/smil/text-node.js#L1) which are required by [Smil](https://github.com/dariocravero/readium-js/blob/master/src/sdk/models/smil/index.js#L29-L30).